CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing
+
+## Submitting Code To The Playbooks
+
+Following these guidelines will it easier for people to review your
+code and merge it faster.
+
+
+1. Please add **documentation** to any user-facing changes you make
+
+If you add a new option, or support for a new feature, please describe
+how to use that feature in the relevant README.
+
+2. **Do not rebase** when adding changes to an existing pull request
+
+Make the change as an new commit instead and push it to your branch.
+That way, Github lets us see just the new changes, which will make the
+review process faster.
+
+When you do a rebase, we can't see what changed before the last
+revision and have to look at the entire pull request again -- even if
+all you added was a small documentation fix!
+
+If you need to bring in changes from the master, you can do `git merge
+master` instead or rebasing.
```

Comment:
Rebasing changes and resolving conflicts produces a better looking history than master merges, but I can live with that.